### DISCUSSION TITLE
Ingestion 429 Too Many Requests

### DISCUSSION THREAD

**Discussion by salvorusso** [mayooear/ai-pdf-chatbot-langchain#196] [Q&A]
Any idea why when i run `npm run ingest` does this?

```
creating vector store...
error Error: Request failed with status code 429
```

Is it Pinecone or OpenAI problem? Any solution?

**Comment by baacael** [comment]
I have the same error .
 did you find a solution?

**Reply by guccibase** [reply]
Using a different model solved it for me. Try gpt-3.5-turbo-0301

**Comment by guccibase** [comment]
I have the same issue. Tried everything but it's still not working. I event upgraded to pay as you go in openai but still getting the error. Any solutions?

**Comment by guccibase** [comment]
Using a different model solved it for me. Try gpt-3.5-turbo-0301

**Comment by mlau5689** [comment]
I determined that my chunk size was too large (default: 1000). I reduced to 500, and my ingestion finally worked.
